ark/settings.py

```diff
@@ -21,3 +21,7 @@
                            'major_axis_equiv_diam_ratio', 'convex_hull_resid',
                            'centroid_dif', 'num_concavities']
 REGIONPROPS_MULTI_COMP = ['nc_ratio']
+
+# spatial-LDA minimum required columns
+BASE_COLS = ["point", "label", "cell_size", "centroid-0", "centroid-1",
+             "pixelfreq_hclust_cap", "name"]
```

Comment:
Not suer if we want to use the globals defined in the other settings here, i.e
```
BASE_COLS = [FOV_ID,  CELL_LABEL, ...]
```

Reply:
Yes, lets try and mirror the column names in the segmentation portion of this pipeline. You can rename the columns from the example data table you got from Alea to be in line with what the rest of the repo is using